server/scratchy_server/resources/roomres.py

```diff
@@ -35,7 +35,7 @@ class AllRoomRes(MethodResource):
     @use_kwargs({"userId": fields.String()}, location="query")
     def get(self, userId=None):
         if not userId is None: # Search room by containing a specific userid
-            return {"rooms": RoomModel.objects(users=[ userId ])}
+            return {"rooms": RoomModel.objects(users= userId)}
```

Comment:
backend ? disgusting.

Reply:
@Benvii made that because yesterday can't have 2 users in the same room